src/Microsoft.AspNetCore.Mvc.ViewFeatures/HtmlHelperOfT.cs

```diff
@@ -370,14 +363,20 @@ public string NameFor<TResult>(Expression<Func<TModel, TResult>> expression)
                 htmlAttributes);
         }
 
-        protected string GetExpressionName<TResult>(Expression<Func<TModel, TResult>> expression)
+        /// <summary>
+        /// Gets the name for the lambda expression.
+        /// </summary>
+        /// <typeparam name="TResult">The type of the result.</typeparam>
+        /// <param name="expression">The expression.</param>
+        /// <returns>The expression name.</returns>
+        public string GetExpressionName<TResult>(Expression<Func<TModel, TResult>> expression)
```

Comment:
@pranavkm It is replacement for `ExpressionHelper.GetExpressionText(expression)`, isn't it?

Reply:
Yup.